config/config.sample.php

```diff
@@ -1435,6 +1435,16 @@
 
 /**
  * Hashing
+ */
+
+/**
+ * By default Nextcloud will use the Argon2 password hashing if available.
+ * However if for whatever reason you want to stick with the PASSWORD_DEFAULT
+ * of your php version. Then set the setting to true.
+ */
+'hashingDefaultPassword' => false,
```

Comment:
```suggestion
'hashing_default_password' => false,
```

we use snake_case for *most* config params